### ISSUE TITLE
Only last one works if different classes hook same selector 

### ISSUE THREAD

**Issue by weiczhu** [steipete/Aspects#82]
Only last one works if different classes hook same selector

**Comment by steipete** [comment]
Yes, that's documented. There's a branch where I tried to work on this.

**Comment by mikeRozen** [comment]
What will be best practice to call `viewWillAppear` of which viewController? For Instance If I want to add GoogleAnalytcs for each controller?(I can follow the example )

`[UIViewController aspect_hookSelector:@selector(viewWillAppear:) withOptions:AspectPositionAfter usingBlock:^(id<AspectInfo> aspectInfo, BOOL animated) {
    NSLog(@"View Controller %@ will appear animated: %tu", aspectInfo.instance, animated);
} error:NULL];`

but in that case I will also link some apple controller like `navigationControllers` and so on...
